docs/pages/enroll-resources/agents/add-service-to-agent.mdx

```diff
@@ -0,0 +1,263 @@
+---
+title: Enable a New Service on an Agent
+description: Explains how to edit the services that are running on a Teleport Agent.
+---
+
+A single Teleport Agent can run multiple services. This guide shows you how to
+adjust the services that a Teleport Agent runs so you can proxy different
+resources in your infrastructure.
+
+## How it works
+
+When a Teleport Agent joins a cluster, it presents a **join token** to the
+Teleport Auth Service in order to authenticate itself. A join token is a dynamic
+resource stored on the Teleport Auth Service backend. It contains a time to live
+(TTL), name, and a list of Teleport services that the token authorizes to run on
+the agent. 
+
+When a Teleport Agent joins a cluster, the Teleport Auth Service issues a
+certificate for the agent. The certificate contains the Teleport services that
+it authorizes the agent to run. To run new services on an agent, you must repeat
+the initial join procedure for those services. The agent needs to re-join the
+cluster with the new token and receive a new certificate.
+
+## Prerequisites
+
+- A Teleport cluster
+- At least one Teleport Agent running in your cluster, either on a Linux server
+  or a Kubernetes pod
+ - (!docs/pages/includes/tctl.mdx!)
+
+As an example, this guide shows you how to add a service to an agent using the
+token join method. This is one of multiple available join methods. Read the
+[Using Teleport Agents overview](agents.mdx) for a complete list.
+
+## Step 1/3. Generate a new join token
+
+Generate a new join token for all services running on an agent, including any
+new services you want to run. In our example, we want to create a join token for
+the Teleport Kubernetes Service and Teleport Application Service, so we specify
```

Comment:
it's not entirely clear here which service was the existing and which one we're adding. I'd make that more clear.